third_party/astcenc/Source/mathlib.h

```diff
@@ -27,7 +27,7 @@ float cospi(float p);
 
 float nan(int p);
 
-#if __cplusplus < 201103L
+#if (!_MSC_VER) && (__cplusplus < 201103L)
```

Comment:
should be `#if !defined(_MSC_VER) &&` ...

Reply:
done